stable/rabbitmq-ha/values.yaml

```diff
@@ -398,6 +398,10 @@ persistentVolume:
   ##   GKE, AWS & OpenStack)
   ##
   # storageClass: "-"
+  ## selector can be used to match an existing PersistentVolume
+  # selector:
```

Comment:
I prefer to keep values defaulted to avoid nils. So would suggest `selector: {}` instead of commented out.

Reply:
Fixed